Makefile

```diff
@@ -577,6 +577,11 @@ INTEG_TEST_XDC_DIR=hostxdc
 INTEG_TEST_NDC_ROOT=./host/ndc
 INTEG_TEST_NDC_DIR=hostndc
 
+# INTEG_TEST_ROOT requires to use test flags that are not used in other sub directories
+# go test require all test package support the flags that are used in the test
+# So INTEG_TEST_DIRS contains all test directories of INTEG_TEST_ROOT that are not using test flags
+INTEG_TEST_DIRS=$$(go list $(INTEG_TEST_ROOT)/... | grep -v $(INTEG_TEST_XDC_ROOT) | grep -v $(INTEG_TEST_NDC_ROOT) | grep -v $(INTEG_TEST_ROOT)$$)
+
```

Comment:
The goal makes sense, but I'm not sure this actually achieves what the comment is talking about... seems like it's actually "integration tests that are not multi-host" (xdc and ndc)?

and it's excluding the top level tests, which doesn't seem relevant - there are tests that both use e.g. Cassandra and that do not.  are they getting run anywhere after this?

(that said, I do see that this should just be _adding_ tests, so it seems like an improvement either way)

Reply:
I have a feeling that our `.host` was created with another purpose, however right now it's the main integration test. Propably, I should have created another buildkite job, but at the same time, the start up duration of a builkite job is more than these tests will take. 

Propably `.host` should not rely on test flags and use environments to specify PERSISTENCE TYPE, however I think it should be part of another PR.